spec/project_spec.rb

```diff
@@ -0,0 +1,40 @@
+# spec/project_spec.rb
+require './lib/player'
+require './lib/board'
+
+describe Player do
+  let(:player_test) { Player.new('X', 'player1', [1, 4, 7]) }
+  describe '#winner?' do
+    it 'winner is player 1' do
+      expect(player_test.winner?).to eql(true)
+    end
+  end
+end
+
+describe Board do
+  let(:test_board) { Board.new([1, 2, 'X']) }
+  describe '#grid_filled?' do
+    it 'grid is not completely filled with strings' do
+      expect(test_board.grid_filled?).not_to eql(true)
+    end
+  end
```

Comment:
- [ ] Your specs should cover positive and negative scenarios. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/commonsense.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/)).

> In this example, you need to test with two arrays, one that's filled another that's not completely filled, then test as shown below (this is just an example implementing the positive and negative scenarios, you can do it anyhow you see fit).

```suggestion
  describe '#grid_filled?' do
    it 'returns true if grid is completely filled with strings' do
      expect(filled_test_board.grid_filled?).to eq(true)
    end
    
    it 'returns false if grid is not completely filled with strings' do
      expect(unfilled_test_board.grid_filled?).to eq(false)
    end
  end
```